src/main/java/org/tron/common/runtime/vm/VM.java

```diff
@@ -22,51 +21,32 @@
 import org.tron.common.runtime.vm.program.Stack;
 
 @Slf4j(topic = "VM")
-
 public class VM {
 
-  // private static final Logger logger = LoggerFactory.getLogger("TronVM");
-  private static BigInteger _32_ = BigInteger.valueOf(32);
+  private static final BigInteger _32_ = BigInteger.valueOf(32);
   private static final String logString = "{}    Op: [{}]  Energy: [{}] Deep: [{}]  Hint: [{}]";
 
-  // max mem size which couldn't be paid for ever
-  // used to reduce expensive BigInt arithmetic
-  private static BigInteger MAX_MEM_SIZE = BigInteger.valueOf(Integer.MAX_VALUE);
-
   // 3MB
-  private static BigInteger MEM_LIMIT = BigInteger.valueOf(3 * 1024 * 1024);
-
-
-  /* Keeps track of the number of steps performed in this VM */
-  private int vmCounter = 0;
-  private boolean vmTrace;
-  // private long dumpBlock;
+  private static final BigInteger MEM_LIMIT = BigInteger.valueOf(3 * 1024 * 1024);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Cast one of the operands of this multiplication operation to a "long". ([squid:S2184](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2184))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1513&issues=AWXICm3x_eEZPtUmm6Nn&open=AWXICm3x_eEZPtUmm6Nn)

[ISSUE_KEY]: AWXICm3x_eEZPtUmm6Nn